framework/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Yii Framework 2 Change Log
 - Bug #18845: Fix duplicating `id` in `MigrateController::addDefaultPrimaryKey()` (WinterSilence, samdark)
 - Bug #17119: Fix `yii\caching\Cache::multiSet()` to use `yii\caching\Cache::$defaultDuration` when no duration is passed (OscarBarrett)
 - Bug #18842: Fix `yii\base\Controller::bindInjectedParams()` to not throw error when argument of `ReflectionUnionType` type is passed (bizley)
+- Bug #18880: Fix `yii\helpers\ArrayHelper::toArray()` for `DateTime` objects in PHP >= 7.4 (rhertogh)
```

Comment:
```suggestion
- Bug #18880: Fix `yii\helpers\ArrayHelper::toArray()` for `DateTime` objects in PHP > 5.4 (rhertogh)
```

Reply:
@bizley Before PHP 7.4 the `toArray()` function already returned an array. Please see https://bugs.php.net/bug.php?id=79041 for more information.
Therefore this fix only fixes the behavior for PHP >= 7.4.

Reply:
Oh, right, I understood that feature is for PHP 7.4+ but the actual FIX is for that version. 👍🏻